pyproject.toml

```diff
@@ -20,6 +20,7 @@ classifiers = [
 ]
 requires-python = ">=3.12.0"
 dependencies    = [
+    "chardet>=5.2.0",
```

Comment:
I would like to align with dependencies chosen by the home assistant project, would it be possible to use this one instead https://pypi.org/project/faust-cchardet/